packages/components/src/summary-button/index.tsx

```diff
@@ -65,7 +66,12 @@ function UnforwardedSummaryButton(
 									{ strapline }
 								</Text>
 							) }
-							<Text className="summary-button-title">{ title }</Text>
+							<Text
+								className="summary-button-title"
+								style={ textAlign ? { textAlign } : undefined }
```

Comment:
Won't it be undefined anyways?

```suggestion
								style={ { textAlign } }
```